docs/File-Formats.md

```diff
@@ -1105,58 +1105,44 @@ rank<TAB>gene<TAB>N<TAB>n<TAB>p<TAB>q
 ```
 
 ## Gene Panel Data
-Gene panel information can assign a list of genes that a genetic profile should consist of for a specific sample.
+Gene panel functionality can specify which genes are assayed on a panel and assign samples and genetic profiles (such as mutation data) to a panel.
 
-#### Gene Panel File
-The gene panel file follows the format of a meta file with the following fields:
-1. **stable_id**: The name of the gene panel. This should be unique across all studies, as gene panels can be globally applied to any sample and any genetic profile.
-2. **description**: A descripion of the gene panel.
-3. **gene_list**: Tab separated genes, represented either by all hugo symbols or all entrez_gene_ids.
+To include gene panel data in your instance, the following data and/or configurations can be used:
+1. **Gene panel file**: This file contains the genes on the gene panel. A panel can be used for multiple studies within the instance and should be loaded prior to loading a study with gene panel data. For information on the format and import process please visit: [Import-Gene-Panels](Import-Gene-Panels.md).
+2. **Gene panel matrix file**: This file is used to specify which samples are sequenced on which gene panel in which genetic profile. This required for mutation and fusion data, because the MAF and fusion formats are unable to include samples which are sequenced but no mutations are found. For other genetic data, columns can be added to specify their gene panel, but a property can also be added to their respective meta file, because these data files contain all profiled samples. Although the gene panel matrix functionality overlaps with the case list functionality, a case list for mutations (`_sequenced`) is also required.
+3. **Gene panel property in meta file**: Adding the `gene_panel:` property to the meta file of a non-mutation data file, will assign all samples from that data type to the gene panel. In this case it is not necessary to include a column for this profile in the gene panel matrix file.
```

Comment:
Suggest changing "...of a non-mutation data file, will assign all samples from that data type to the gene panel." to "...of a non-mutation data profile, will assign all samples from that profile to the gene panel.".

Reply:
Done